### PR TITLE
lower w_class of chisels and ashtrays

### LINKED ISSUE #9741: Ash trays shouldn't be normal sized items
There are a lot of items that should be considered too big for pockets, but fit anyways. Ashtrays aren't much of an item, so why not let them be considered small or tiny just for the sake of being able to carry one in your pocket?

### LINKED ISSUE #9961: Chisel size
(WEB REPORT BY: pwirby REMOTE: 172.93.109.202:7777)

> Revision (Should be above if you're viewing this from ingame!)
> Take a chisel, examine it/ try to put it in your pocket/box.
> 
> General description of the issue
> The chisel is a normal sized item but it's IRL more a pocket one. Also, it has no use other than engrave so making it pocket sized make it much useful.
> 
> What you expected to happen

To be pocket sized.

> What actually happened

Is normal sized.

> Steps to reproduce if possible

Take a chisel, examine it/ try to put it in your pocket/box.

### PR DESCRIPTION
**Pull request by unid15**
Fixes #9961 (chisels now have 2 w_class)
Fixes #9741 (ashtrays now have 1 w_class)

### CODE REVIEW

File: code/WorkInProgress/Chinsky/ashtray.dm [hunk 1]
```diff
@@ -1,12 +1,13 @@
 /obj/item/ashtray
 	icon = 'icons/ashtray.dmi'
-	var/
```
Comment (9600bauds) [inline code/WorkInProgress/Chinsky/ashtray.dm:3]: holy oldcode batman

**Comment by Intigracy** [comment]
Since you're fixing bad formatting, can you fix the die() 's being partial pathed?

https://github.com/d3athrow/vgstation13/pull/9975/files#diff-b83e9988ba73c7700ce6520961a7c2c3R106

**Comment by Imagebard** [comment]
>PR to modify two obscure rarely used items
Needs Final Decision

**COMIIIIIIIIIIIIIIIIIC**

**Comment by Zth--** [comment]
Why? Stuff like this should be instamerged...

**Comment by 9600bauds** [comment]
I'm less concerned with NFD and more concerned with the fact that it stayed up for 4 days with NFD

**Comment by unid15** [comment]
dang I forgot to pay the monthly fee of 20 pomfcoins for faster merge times

**Comment by clusterfack** [comment]
If it hasn't been obvious I've had an extremely busy week personally

**Comment by 9600bauds** [comment]
Have a good weekend

**Comment by PJB3005** [comment]
@Imagebard NFD basically means "ready for merge" not "controversial pomf needs to decide"

**Comment by Zth--** [comment]
NFD means it was discussed (it maybe was controversial), and know it needs... A FINAL DECISION.
I don't think we need a "ready for merge" tag, we're bleeding edge. Just merge it.

**Comment by Imagebard** [comment]
Okay ignoring all the confusion in here...
Last time I checked(It probably changed and PJB3005 is probably right on this one), "Needs Final Decision" was usually a tag people put up to signal that the PR was ready but the maker didn't want to merge it.

So basically, unid15 changed the w-size of two obscure items. It was up for 1 day, then ComicIronic placed it on NFD for 4 days. So basically we waited 5 days to merge this little PR even though a coder already checked it.

**Comment by clusterfack** [comment]
There is literally no reason to rush these things

**Comment by Intigracy** [comment]
There is literally no reason to rush anything that's not an important
bug/exploit fix.

Cf's spam merging da bes.

**Comment by Imagebard** [comment]
_11 lines_

**Comment by clusterfack** [comment]
And? Why should that mean it is rushed?

Who gains if its rushed? No one
Who loses if its rushed? Everyone

**Comment by ihadtoregisterforthis** [comment]
Would you believe we have a guide with the tags?
https://github.com/d3athrow/vgstation13/wiki/Tagging-guidelines

**Comment by Imagebard** [comment]
The rare clusterfack codertype would be the most dangerous coder in the world

But only if the world slowed _waaaaaaay_ the fuck down

**Comment by clusterfack** [comment]
If everyone signs an agreement to never complain when something is merged with bugs or something was merged that they don't like or didn't get to comment upon, THEN we'll merge everything within a day willy-nilly.

But until that day, no we aren't. And when the people who complain about those things simultaneously complain about merge times, then I just shake my head.

**Comment by Imagebard** [comment]
5 day long NDF on a 11 line-long PR. Its the _only_ way.

(No, I'm not going to entertain this specific argument).

**Comment by clusterfack** [comment]
The small size merely means it has a far less likely chance to have conflicts even

**Comment by Intigracy** [comment]
Things don't get merged every day fam.

Clusterfack specifically will merge a large amount of ready PR's at once,
regardless of size.

I've had small grammar fixes stay up for days at a time but it _doesn't
matter_.

**Comment by 9600bauds** [comment]
desu it would be nice to have #9950 merged so I can stop fucking committing and switching branches to test my shit

**Pull request merged**